### PR TITLE
Update German translation

### PR DESCRIPTION
**Pull request by psjnek**
Acceptance Criteria:

- [x] Features and enhancements must be fully implemented so that they can be released at any time without additional work
- [x] Automated unit and/or acceptance tests are mandatory to ensure the changes work as expected and to reduce repetitive manual work
- [x] Frontend components must be responsive to work and look properly on phones, tablets, and desktop computers; you must have tested them on all major browsers and different devices
- [x] Documentation and translation updates should be provided if needed
- [x] In case you submit database-related changes, they must be tested and compatible with SQLite 3 and MariaDB 10.5.12+

### CODE REVIEW

**Comment by lastzero** [comment]
Please note that translations should only be edited with Weblate:

https://translate.photoprism.app/projects/photoprism/frontend/de/

Thank you for your contribution!

**Comment by psjnek** [comment]
> Please note that translations should only be edited with Weblate:
> 
> https://translate.photoprism.app/projects/photoprism/frontend/de/
> 
> Thank you for your contribution!

Ok, should I close this issue now or could you maybe accept it anyway?

File: frontend/src/locales/de.po [hunk 1]
```diff
@@ -968,7 +968,7 @@ msgstr "Favoriten"
 
 #: src/page/about/feedback.vue:7 src/options/options.js:439
 msgid "Feature Request"
-msgstr "Feature Request"
+msgstr "Feature-Anfrage"
```
Comment (lastzero) [inline frontend/src/locales/de.po]: [Feature-Request](https://de.wikipedia.org/wiki/Feature-Request) oder Verbesserungsvorschlag sagt die deutsche Wikipedia. Alternativ auch "Leistungsmerkmalanforderung", aber das würden wir definitiv nicht mergen :)

Comment (psjnek) [reply frontend/src/locales/de.po]: "Verbesserungsvorschlag" klingt gut.

**Comment by psjnek** [comment]
> Please note that translations should only be edited with Weblate:
> 
> [translate.photoprism.app/projects/photoprism/frontend/de](https://translate.photoprism.app/projects/photoprism/frontend/de/)
> 
> Thank you for your contribution!

Ich habe in CONTRIBUTING.md geguckt, womit ihr Photoprism übersetzt, jedoch wurde dort Weblate nicht genannt. Soll ich einen Hinweis hinzufügen, dass Übersetzungen nur durch Weblate geschehen sollen?

**Comment by lastzero** [comment]
Yes, it might be good to mention this there as well, although it is already covered in our developer guide (and linked in the footer of our website):

https://docs.photoprism.app/developer-guide/translations-weblate/

**Comment by lastzero** [comment]
I'll be happy to merge your changes once you've signed [our CLA](https://www.photoprism.app/oss/faq#cla). Weblate should pick them up and merge them, unless there are conflicts.

**Comment by psjnek** [comment]
> I'll be happy to merge your changes once you've signed [our CLA](https://www.photoprism.app/oss/faq#cla). Weblate should pick them up and merge them, unless there are conflicts.

I just signed it.

**Pull request merged**